src/lib/helpers/config.ts

```diff
@@ -17,31 +19,65 @@ export const ConfigSchema = z.object({
         ),
     }).strict().default({}),
     cache: z.object({
-        enabled: z.boolean().default(true),
-        directory: z.string().default("/var/tmp"),
+        enabled: z.boolean().default(
+            Deno.env.get("CACHE_ENALBED") === "false" ? false : true,
+        ),
+        directory: z.string().default(
+            Deno.env.get("CACHE_DIRECTORY") || "/var/tmp",
+        ),
     }).strict().default({}),
     networking: z.object({
-        ump: z.boolean().default(false),
+        ump: z.boolean().default(
+            Deno.env.get("NETWORKING_UMP") === "true" || false,
+        ),
         proxy: z.string().nullable().default(Deno.env.get("PROXY") || null),
         fetch: z.object({
-            timeout_ms: z.number().optional(),
+            timeout_ms: z.number().optional().default(
+                Number(Deno.env.get("NETWORKING_FETCH_TIMEOUT_MS") || null),
+            ),
```

Comment:
this makes the value change to `0`, which technically works (because of a loose ternary check on `timeout_ms`) but is misleading. Probably better to use:

```suggestion
            timeout_ms: z.number().nullable().default(
                Number(Deno.env.get("NETWORKING_FETCH_TIMEOUT_MS")) || null,
            ),
```
Which will make `timeout_ms` equal `null` by default, which is a bit more explicit.

Reply:
Done, you can modify it because of #106.
Whatever you think is better